### PR TITLE
The third Software Carpentry Workshop in Korea.

### PR DESCRIPTION
**Pull request by statkclee**
The third Software Carpentry Workshop in Korea.

### CODE REVIEW

File: blog/2015/03/first-software-carpentry-korea.html [hunk 1]
```diff
@@ -2,6 +2,28 @@
 layout: blog
 root: ../../..
 author: Victor (Kwnagchun) Lee
+<<<<<<< HEAD
```
Comment (r-gaia-cs) [inline blog/2015/03/first-software-carpentry-korea.html]: @statkclee You need to fix this conflicts. Can you let me know when you did it so I can review it? If you need assistance to solve the conflicts, let us know.

Comment (statkclee) [inline blog/2015/03/first-software-carpentry-korea.html]: @r-gaia-cs, I tried to fix the conflicts above. Sorry for confusion.

File: blog/2015/04/korean-translation-of-software-carpentry-ver-5-2.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+layout: blog
+root: ../../..
+author: Victor (Kwnagchun) Lee
+title: "Korean Translation of Software Carpentry - version 5.2"
+date: 2015-04-16
+time: "13:00:00"
+category: ["Translation", "Version 5.2", "Korea", "eBook"]
+---
+<!-- start excerpt -->
+<p> Based on Software Carpentry version 5.2, we translated <a href="http://www.xwmooc.net/swc/">all the lessons</a> in the form of pdf, html, mobi, epub, and azw3. It took almost 6 months: the first half spent on mostly lessons, and the second half spent on remainings. While translating and building various ebook formats, the original ebook production software of Software Carpentry seemed not to have upfront architectural concerns in terms of Chinese, Japanese, and Korean (aka, CJK)characters problems. Also, we are wondering how to generate various ebook formats from the current developing lessons.
```
Comment (r-gaia-cs) [inline blog/2015/04/korean-translation-of-software-carpentry-ver-5-2.html]: > the original ebook production software of Software Carpentry seemed not to have upfront architectural concerns in terms of Chinese, Japanese, and Korean (aka, CJK)characters problems.

Sorry for this. My knowledge of CKJ is zero.

File: blog/2015/04/korean-translation-of-software-carpentry-ver-5-2.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <li> <a href="http://www.xwmooc.net/swc/book.mobi">MOBI</a> </li>
+      <li> <a href="https://github.com/statkclee/xwmooc-sc/tree/book">Software Carpentry in Korean (Ver 5.2)</a> </li>
+      <li> <a href="https://github.com/swcarpentry/bc">Software Carpentry in English (Ver 5.2)</a> </li>      
+    </ol>
+  </ul>
+
+<h3>What we found and the next...</h3>
+<p>The original ebook production software did not support CJK languages such that we need to tweak in order to make them properly rendered. We have some concerns in terms of the current lessons. We could not find any explicit commitment to the various ebook formats support. We follow other software and associated licenses, but we thought that contents license(creative commons license) need to be modified. Therefore we enforced CC-BY-SA instead of CC-BY only for Korean translated pieces. We hope many people in Korea open and share these excellent contents alike. By the way, we are excited to use fully translated materials with various formats. We look forwards to <a href="statkclee.github.io/2015-04-29-pangyo">the second workshop</a>.
```
Comment (r-gaia-cs) [inline blog/2015/04/korean-translation-of-software-carpentry-ver-5-2.html]: Sorry for did not find time for clean my patch to generate PDF, EPUB, ... Maybe you can share yours with us. =)

**Comment by r-gaia-cs** [comment]
@gvwilson Can you merge this manually?

**Comment by gvwilson** [comment]
Is this one ready to go?

**Comment by r-gaia-cs** [comment]
@gvwilson I think it is but need to merge manually.

**Comment by gvwilson** [comment]
OK, I'll do the merge.

**Pull request merged**